devtools/Makefile

```diff
@@ -1,11 +1,19 @@
 build:
+	${CARGO} +${NIGHTLY_TOOLCHAIN} build ${BUILD_TYPE_FLAG} -p test-runner-client
+	${CARGO} +${STABLE_TOOLCHAIN} build ${BUILD_TYPE_FLAG} -p test-runner-server
 
 check:
+	${CARGO} +${NIGHTLY_TOOLCHAIN} check ${BUILD_TYPE_FLAG} -p test-runner-client
```

Comment:
The `+` option only works for a `cargo` installed with `rustup`. With a basic Fedora install, that does not work. I'm looking into how to address that.

Reply:
```
rustup automatically determines which [toolchain](https://rust-lang.github.io/rustup/concepts/toolchains.html) to use when one of the installed commands like rustc is executed. There are several ways to control and override which toolchain is used:

- A [toolchain override shorthand](https://rust-lang.github.io/rustup/overrides.html#toolchain-override-shorthand) used on the command-line, such as cargo +beta.
- The RUSTUP_TOOLCHAIN environment variable.
- A [directory override](https://rust-lang.github.io/rustup/overrides.html#directory-overrides), set with the rustup override command.
- The [rust-toolchain.toml](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file) file.
- The [default toolchain](https://rust-lang.github.io/rustup/overrides.html#default-toolchain).
```
How about the second and third methods above?

Reply:
@c3d anymore comments about this PR? Or could we move to merge it and improve it later?